### DISCUSSION TITLE
I can't use this app anymore

### DISCUSSION THREAD

**Discussion by rinldo** [noah-nuebling/mac-mouse-fix#1237] [Questions]
Hi all,
About a month ago, after I restarted my mac book m1, I found I could not turn of this app anymore. (attached screenshot, it always in a turn off state even if I click the turn on button)
I restarted my mac again and again but without help.
Does any one how to fix?
Thank you.

Mac os version: 13.6.7（22G720）
Mac mouse fix version: 3.0.3

**Comment by noah-nuebling** [comment]
Hey @rinldo, thanks for reaching out, and I'm sorry for the late reply. Please send me [an email](https://redirect.macmousefix.com/?target=mailto-noah) and I can give you a refund. I can also investigate the problem further, in case you'd like to provide more details.
